mmdet/core/bbox/match_costs/match_cost.py

```diff
@@ -357,3 +357,42 @@ def __call__(self, cls_pred, gt_labels):
             raise NotImplementedError
 
         return cls_cost * self.weight
+
+
+@MATCH_COST.register_module()
+class MaskCost:
+    """MaskCost.
+
+    Args:
+        weight (int | float, optional): loss_weight
```

Comment:
optional here is strange, `weight` need be float or int. Add defaluts value. same as `use_sigmoid`